java/src/main/java/ai/rapids/cudf/ColumnView.java

```diff
@@ -1296,6 +1297,59 @@ public ColumnVector castTo(DType type) {
     return new ColumnVector(castTo(getNativeView(), type.typeId.getNativeId(), type.getScale()));
   }
 
+  /**
+   * This method takes in a nested type and replaces its children with the given views
+   * Note: Make sure the numbers of rows in the leaf node are the same as the child replacing it
+   * otherwise the list can point to elements outside of the column values.
```

Comment:
nit: It would be nice to add in a warning about the return type being a ColumnView and that it does not own the data, so the everything passed into this must out live it.  But that is a general warning about all ColumnViews which is why it is a nit.